release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -579,6 +579,16 @@ With this release, you can monitor updates for individual nodes as a Technology
 [id="ocp-4-15-nodes"]
 === Nodes
 
+[id="ocp-4-15-nodes-icsp-idms-compatibility"]
+==== ICSP, IDMS, and ITMS are now compatible
+
+`ImageContentSourcePolicy` (ICSP), `ImageDigestMirrorSet` (IDMS), and `ImageTagMirrorSet` (ITMS) objects now function in the same cluster at the same time. Previously, in order to use the newer IDMS or ITMS objects, you needed to delete any ICSP objects. Now, you can use any or all of the three types of objects to configure repository mirroring after the cluster is installed. For more information, see xref:../post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror_post-install-preparing-for-users[Understanding image registry repository mirroring].
+
+[IMPORTANT]
+====
+Using an ICSP object to configure repository mirroring is a deprecated feature. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
```

Comment:
Will/might , "Might" would tie us in less knots if a feature were somehow, although rare, brought back to life.

Reply:
@dfitzmau Good thought, but "will" seems to be the standard wording in our docs. Likely would want to do a global find/replace...